media/css/firefox/all.less

```diff
@@ -276,6 +276,7 @@ body.nightly {
     }
 
     #language-search {
+        width: @widthTablet - (@gridGutterWidth * 2 * 2.25);
```

Comment:
The width for language search is set here at tablet viewport: https://github.com/mozilla/bedrock/blob/master/media/css/firefox/all.less#L278-L297.

That certainly needs fixing too (nice spot), but my comment review comment still applies even after fixing this.